### DISCUSSION TITLE
Welcome to AutoTrain Discussions!

### DISCUSSION THREAD

**Discussion by abhishekkrthakur** [huggingface/autotrain-advanced#258] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DimIsaev** [comment]
how to create a dataset (format) for fine tuning llm

**Comment by zufeshan12** [comment]
Are you planning to add Object Detection support in autotrain platform soon? Right now only Image classification from computer vision could be seen as present in the list of tasks.

**Comment by abhishek1297** [comment]
What would be the default parallelism used in the case of Multi-GPUs? If my default accelerate setup is 
```
- GPU type: NVIDIA A100-PCIE-40GB
- `Accelerate` default config:
        - compute_environment: LOCAL_MACHINE
        - distributed_type: MULTI_GPU
        - mixed_precision: no
        - use_cpu: False
        - num_processes: 2
        - machine_rank: 0
        - num_machines: 1
        - rdzv_backend: static
        - same_network: False
        - main_training_function: main
        - downcast_bf16: False
        - tpu_use_cluster: False
        - tpu_use_sudo: False
```

and I fine-tune **Llama2-13b-chat-hf** using
`CUDA_VISIBLE_DEVICES=0,1 autotrain llm train ...`

I cannot find any configuration regarding Multi-GPU training in the model files or tensorboard logs.

**Comment by akajeevan** [comment]
Hi, tried using AutoTrain Dreambooth on Google Colab but it's not able to run. 

Here's the Error, I get while running it:

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
lida 0.0.10 requires kaleido, which is not installed.
llmx 0.0.15a0 requires cohere, which is not installed.
llmx 0.0.15a0 requires openai, which is not installed.
tensorflow-metadata 1.14.0 requires protobuf<4.21,>=3.20.3, but you have protobuf 4.23.4 which is incompatible.
tensorflow-probability 0.22.0 requires typing-extensions<4.6.0, but you have typing-extensions 4.9.0 which is incompatible.
[ ]

To be honest, not very familiar with using Google Colab and coding, so find it tough but hope to learn. 

What should I do to get the AutoTrain Dreambooth to be running ? 

Would appreciate if any of the community member can guide me in successfully run the Programme. Thx

**Reply by abhishekkrthakur** [reply]
just ignore and continue. this has been discussed several times in issues.

**Reply by akajeevan** [reply]
Thx for the quick response, will ignore it.

**Comment by Stighebbelstrup** [comment]
I am trying to use the script for DreamBooth script in aws notebook with prior-preservation using the xl model. I could not get this to work before adding this to the xl.config in the __main__.py "prior_loss_weight = config.prior_loss_weight" since this parameter seems to be missing in the configs for this model. Perhaps also for other models but I have not tested others than the xl model.
